docs/API.md

```diff
@@ -1806,6 +1806,19 @@ A delegatable macaroon will only be returned to an authorized user (not
 including admin). It will carry the same privileges as the macaroon used
 to authorize the request.
 
+#### GET /whoami
+
+This endpoint returns a user name and list of groups in JSON format.
```

Comment:
This endpoint returns the user name of the client and the list of groups the user
is a member of. This endpoint requires authorization.

?

(no need to mention JSON format - it's implicit in all the endpoints in this doc)